sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -127,10 +127,35 @@ def config_checks(self):
                 "Autodiscovery is disabled, autodiscovery_include and autodiscovery_exclude will be ignored"
             )
 
+    def split_sqlserver_host_port(self, host):
+        """
+        Splits the host & port out of the provided SQL Server host connection string, returning (host, port).
+        """
+        if not host:
+            return host, None
+        host_split = host.split(',')
+        if len(host_split) == 1:
+            return host_split[0], None
+        if len(host_split) == 2:
+            return host_split
+        # else len > 2
+        s_host, s_port = host_split[0:2]
```

Comment:
I suggest adding a `.trim()` onto each of these and the one above, as someone will inevitably put a space between the comma and port.

Reply:
good point, updated